### DISCUSSION TITLE
Utilising the DeepLesion data set

### DISCUSSION THREAD

**Discussion by YipengHu** [DeepRegNet/DeepReg#597] [Ideas]
Paper: https://www.spiedigitallibrary.org/journals/journal-of-medical-imaging/volume-5/issue-03/036501/DeepLesion--automated-mining-of-large-scale-lesion-annotations-and/10.1117/1.JMI.5.3.036501.full?SSO=1

Data: https://nihcc.app.box.com/v/DeepLesion

**Comment by mathpluscode** [comment]
What's the objective here, build a new model?

**Comment by YipengHu** [comment]
the classical registration is the baseline method for aligning intra-patient lesions for this dataset.

**Reply by mathpluscode** [reply]
ok, then I guess adding classical registration methods can be a new feature to add :)
